index.Rmd

```diff
@@ -43,3 +43,4 @@ _Programming skills_
 ## AnVIL Collection {-}
 
 Please check out our full collection of AnVIL resources below!
+
```

Comment:
Maybe you want to take this out, since you aren't rendering the "AnVIL Collection"?